src/query/shorthand.ts

```diff
@@ -134,6 +134,13 @@ export function spec(specQ: SpecQuery,
     }
   }
 
+  if (!!specQ.width) {
```

Comment:
I think you have to take `include` into account here too.  

(I also notice that we forgot to do the same for `transform` above -- please help fix that too -- maybe in a separate PR.)